### ISSUE TITLE
Can't include export keyword in comments.

### ISSUE THREAD

**Issue by xiaoxiangmoe** [developit/microbundle#253]
see [microbundle-error](https://github.com/xiaoxiangmoe/microbundle-error)

```ts
// FIXME: this is a bug of micro bundle.
// 1. only can ex🦀️port default value by named ex🦀️port.
// 2. and can't include ex🦀️port keyword in comments.
export { identity as default };
```
If I remove `🦀️`, it will throw an error.
If I change it to `export default identity`, it will throw an error.

**Comment by developit** [comment]
Here's the offending lines:

https://github.com/developit/microbundle/blob/master/src/index.js#L297-L302

**Comment by Andarist** [comment]
IMHO we should switch entirely to `exports: 'named'` or work with webpack to fix https://github.com/webpack/webpack/issues/7973 . `exports: 'auto'` atm is harmful for webpack consumers.

**Comment by developit** [comment]
The stripping mode I had added is really only designed for cases like this:

https://github.com/developit/preact/blob/master/src/preact.js#L12-L32

I think we could make it an option or use an AST pass (via a rollup plugin) to detect that pattern and avoid the double export for cjs/umd output targets.

**Comment by ForsakenHarmony** [comment]
I think this should be fixed with #265
